cmd/clusterctl/cmd/move.go

```diff
@@ -64,13 +75,27 @@ func init() {
 		"The namespace where the workload cluster is hosted. If unspecified, the current context's namespace is used.")
 	moveCmd.Flags().BoolVar(&mo.dryRun, "dry-run", false,
 		"Enable dry run, don't really perform the move actions")
+	moveCmd.Flags().BoolVar(&mo.backupToFolder, "to-folder", false,
+		"Write Cluster API objects and all dependencies from a management cluster to folder.")
+	moveCmd.Flags().BoolVar(&mo.restoreFromFolder, "from-folder", false,
+		"Restore Cluster API objects from folder.")
+	moveCmd.Flags().StringVar(&mo.directory, "directory", "",
+		"The directory to save Cluster API objects to as yaml files")
+
+	// Flags mutually exclusive
+	moveCmd.MarkFlagsMutuallyExclusive("from-folder", "kubeconfig")
+	moveCmd.MarkFlagsMutuallyExclusive("to-folder", "to-kubeconfig")
+	moveCmd.MarkFlagsMutuallyExclusive("from-folder", "to-folder")
 
 	RootCmd.AddCommand(moveCmd)
 }
 
 func runMove() error {
 	// if no to kubeconfig provided and it's not a dry run, return error
-	if mo.toKubeconfig == "" && !mo.dryRun {
+	if !mo.backupToFolder &&
+		!mo.restoreFromFolder &&
+		mo.toKubeconfig == "" &&
+		!mo.dryRun {
 		return errors.New("please specify a target cluster using the --to-kubeconfig flag")
```

Comment:
This eroor message needs to be elaborated. A target is not always a `--to-kubeconfig`.